configs/vuetifyjs.json

```
@@ -48,7 +49,8 @@
     }
   },
   "custom_settings": {
-    "attributesForFaceting": ["language"]
+    "attributesForFaceting": ["language"],
```

```suggestion
    "attributesForFaceting": ["lang"],
```

It should match the `variables` name